src/base/static/libs/quill-image-resize.js

```diff
@@ -0,0 +1,273 @@
+/**
```

Comment:
I see. Have you commented your changes (i.e. put in a note that they are monkey patches)? I would suggest doing that.